tests/integration/users.test.js

```diff
@@ -1508,20 +1512,21 @@ describe('DELETE /users/:id', () => {
       expect(response.statusCode).toBe(403);
     });
 
-    it('should return 403 if user is registered to course', async () => {
+    it('should return 403 if user has activity histories', async () => {
```

Comment:
Que penses-tu de verifier qu'on a bien une 200 si l'utilisateur est juste inscrit a une formation sans historique ?

Reply:
c'est le cas du test should delete user, il a une companyrequestlink et est inscrit à une activité (cf seeds)